contivmodel/contivModel.go

```diff
@@ -3457,7 +3457,7 @@ func ValidateNetprofile(obj *Netprofile) error {
 		return errors.New("bandwidth string invalid format")
 	}
 
-	if obj.Burst > 10486 {
+	if obj.Burst > obj.Burst {
```

Comment:
How would this condition ever be true? `x` can't be greater than `x`

Reply:
It's ensure that some errors, I will be commit again. sorry.

Reply:
I reverted the change on `master`.  Feel free to open a new PR 👍